packages/vue-components/src/Dropdown.vue

```diff
@@ -6,7 +6,7 @@
   >
     <slot name="button">
       <a
-        class="dropdown-toggle"
+        class="dropdown-toggle nav-link"
```

Comment:
This component is also used by the [`navbar`](https://deploy-preview-2038--markbind-master.netlify.app/userguide/components/navigation#navbars) to create dropdowns.

As `dropdown` component in `navbar` already has `nav-link` class, the padding is applied here twice.

Notice how the links are no longer aligned on the navbar due to having extra paddings.
![image](https://user-images.githubusercontent.com/47470981/202355978-753a7c1f-f623-4690-b06f-f542f3a7e467.png)

![image](https://user-images.githubusercontent.com/47470981/202356051-e423255b-1104-458e-a89c-ccbd4725d7c9.png)

The relevant code:
```html
<navbar type="primary">
  <!-- Brand as slot -->
  <a slot="brand" href="/" title="Home" class="navbar-brand">MarkBind</a>
  <li><a href="/userGuide/components/navigation.html#navbars" class="nav-link">Highlighted Link</a></li>
  <!-- You can use dropdown component -->
  <dropdown header="Dropdown" class="nav-link">
    <li><a href="#navbars" class="dropdown-item">Option</a></li>
  </dropdown>
  <!-- For right positioning use slot -->
  <li slot="right">
    <a href="https://github.com/MarkBind/markbind" target="_blank" class="nav-link">Fork...</a>
  </li>
</navbar>
```

Reply:
Ahhh yes this is an issue, let me see if I can find a workaround for this 👍

Reply:
hi @jonahtanjz, I think it is okay to simply remove the `nav-link` classname from the dropdown element to remove the extra padding. i.e.

```html
<navbar type="primary">
  <!-- Brand as slot -->
  <a slot="brand" href="/" title="Home" class="navbar-brand">MarkBind</a>
  <li><a href="/userGuide/components/navigation.html#navbars" class="nav-link">Highlighted Link</a></li>
  <!-- You can use dropdown component -->
  <dropdown header="Dropdown"> <!-- class="nav-link" removed -->
    <li><a href="#navbars" class="dropdown-item">Option</a></li>
  </dropdown>
  <!-- For right positioning use slot -->
  <li slot="right">
    <a href="https://github.com/MarkBind/markbind" target="_blank" class="nav-link">Fork...</a>
  </li>
</navbar>
```

the dropdown element element already wraps its anchor element with a `nav-link` anyway so I think the outer `nav-link` is safe to remove. I think this way the navbar elements would also be more consistent as the classname is only applied to the anchor elements within the navbar elements, dropdown or not:

<img width="253" alt="image" src="https://user-images.githubusercontent.com/39848410/202435818-1de842e5-f747-4baa-9df8-ace90b4a8b15.png">

I noticed that this would also affect the text colour of the anchor element and have accounted for it with another commit.

does this work fix, or is there a better approach?

Reply:
I think this approach is fine but one main concern is that currently, most of the MarkBind sites will have the `nav-link` class in the `dropdown` component. If we go with this approach, users will have to remove the class on their end in the next release, which will be considered sort of a "breaking" change in the UI.

One workaround that you can do for now is to have an overriding CSS within `Dropdown.vue` to remove the padding from `nav-link` on the outer `<li>` element. This should ensure that there will not be extra padding with or without the `nav-link` class on the `dropdown` component. Once we are ready to make a release with breaking changes, we can then remove this workaround.

Note: You will need to update the user guide as well under [`navbar`](https://deploy-preview-2038--markbind-master.netlify.app/userguide/components/navigation#navbars) to remove the class.

Reply:
I'm not sure if removing the nav-link class is the best way to go. Because that means that it won't use the bootstrap styling anymore and may not be consistent with the other elements that use the Bootstrap styling. 

Should I still proceed to remove it anyway? @jonahtanjz

Reply:
![image](https://user-images.githubusercontent.com/77204346/224489366-d3f55dbb-e51c-47a1-89e0-5427dd009fce.png)
This is how tabs look like with the removal of the nav-link class from dropdown looks. Some styling can obviously be done but not sure it will be better than using the in built bootstrap